lib/device_api/android/device.rb

```diff
@@ -141,6 +149,11 @@ def package_name(apk)
         result
       end
 
+      def list_installed_packages
+        packages = ADB.pm(serial, 'list packages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.